src/api/xxAMIDOxx.xxSTACKSxx.API/Controllers/Item/DeleteMenuItemController.cs

```diff
@@ -31,10 +32,9 @@ public DeleteMenuItemController(ICommandHandler<DeleteMenuItem, bool> commandHan
         /// <param name="itemId">Id for Item being removed</param>
         /// <response code="204">No Content</response>
         /// <response code="400">Bad Request</response>
-        /// <response code="401">Unauthorized, Access token is missing or invalid</response>
-        /// <response code="403">Forbidden, the user does not have permission to execute this operation</response>
         /// <response code="404">Resource not found</response>
         [HttpDelete("/v1/menu/{id}/category/{categoryId}/items/{itemId}")]
+        [Authorize]
```

Comment:
this will break the FE, I don't think they are passing access tokens in the calls, can you create a separate issue to track this
This should be the standard though

Reply:
I've implemented a custom authorization provider (ConfigurableAuthorizationPolicyProvider) to get around this issue. If JWT bearer authentication is disabled it ignores the [Authorize] attribute and just authorizes the request. If JWT bearer authentication is enabled it authorizes the request using the provided (or default) policy.